apps/docs/src/app/core/component-docs/avatar/avatar-docs.component.html

```diff
@@ -132,3 +132,33 @@
     <fd-avatar-borders-example></fd-avatar-borders-example>
 </component-example>
 <code-example [exampleFiles]="avatarBorders"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="default-image-alter-options" componentName="avatar">
+    Default image with alter icon options
+</fd-docs-section-title>
+<description>
+    To provide alternative icon options by priority use the <code>[alterIcon]</code> property. <br>
+    There's few options can be set the order priority, Avatar component will process them and display first successful option if <code>[image]</code> url fails to load: 
+    <ul>
+        <li>
+            <code>contet</code> - Avatar will display first letter from string, provided as component content
+        </li>
+        <li>
+            <code>alt</code> - Avatar will display first letter from string, provided as <code>alt</code> attribute
+        </li>
+        <li>
+            <code>custom</code> - Avatar will display image, provided as <code>defaultImage</code>
```

Comment:
This part is confusing, let's have it be <code>backupImage</code> - Avatar will display image, provided as <code>backupImage</code>

Reply:
Done